api/src/main/proto/spine/protodata/value.proto

```diff
@@ -166,3 +171,19 @@ message MapValue {
         Value value = 2 [(required) = true];
     }
 }
+
+// A reference to a value in the field specified in the [target] property.
+//
+// Typically such a reference is obtained by parsing an option of a "source" field.
+//
+message Reference {
+
+    // The type of the referenced value.
+    //
+    // If the reference is obtained by parsing a "source" field option, it is the type of the field.
```

Comment:
I don't fully get this sentence.

Did you mean "If ..., it is the type of the **referenced** field."? But then what if not?